### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by victor88121**

### CODE REVIEW

File: file.go
Comment (evanfuller) [inline file.go:59]: `io.Reader` does not need to be closed in right? unless there's some advantage to us changing the interface to `io.ReadCloser`, not sure

**Review by evanfuller — COMMENTED** [review]
these can be addressed now or in follow-up tickets if you prefer but we should add:
1. README
2. LICENSE
3. unit tests for webhook, and maybe some on `ScanFile` that exercise `doChunkedUpload` if possible

**Comment by evanfuller** [comment]
it's probably also worth adding CODEOWNERS

and we'll want to lock down the permissions on this repo under Settings for standard things -- e.g. "require approval before merge to master" and "only allow squash merges" and stuff like that

File: file.go
Comment (victor88121) [reply file.go:59]: it does need to be closed, but i feel like closing is the caller's responsibility, similar to how it is with filesage

**Review by evanfuller — APPROVED** [review]
maybe add a unit test for webhook.go, otherwise lgtm

**Comment by victor88121** [comment]
> maybe add a unit test for webhook.go, otherwise lgtm

will add in another PR

**Pull request merged**
